hermeto/core/package_managers/cargo/main.py

```
@@ -95,9 +104,32 @@ def purl(self) -> PackageURL:
+            # This can happen to some Rust dependencies for Python project,
+            # e.g. cryptography-cffi@0.1.0. This is not a local package, those
+            # are handled elsewhere.
+            return True
+        if self.source.startswith("git+"):
+            return False
+        return "crates.io" in self.source
+
     def to_component(self) -> Component:
         """Convert CargoPackage into SBOM component."""
-        return Component(name=self.name, version=self.version, purl=self.purl.to_string())
+        ref_rest = dict(type=PROXY_REF_TYPE, comment=PROXY_COMMENT)
```

Ruff (0.16.0)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)